src/Presentation/Nop.Web/Areas/Admin/Views/Product/ProductTags.cshtml

```diff
@@ -75,6 +87,28 @@
                                 //return false to don't reload a page
                                 return false;
                             });
+
+                            $('#delete-selected-action-confirmation-submit-button').bind('click', function () {
+                                var postData = {
+                                    selectedIds: selectedIds
+                                };
+                                addAntiForgeryToken(postData);
+                                $.ajax({
+                                    cache: false,
+                                    type: "POST",
+                                    url: "@(Url.Action("ProductTagsDelete", "Product"))",
+                                    data: postData,
+                                    error: function (jqXHR, textStatus, errorThrown) {
+                                        $('#deleteSelectedFailed-info').text(errorThrown);
+                                        $('#deleteSelectedFailed').click();
```

Comment:
You must add a tag helper "nop-alert" for this code to work. Add after <script>: `<nop-alert asp-alert-id="deleteSelectedFailed" />`

Reply:
@DmitriyKulagin done.